app/models/task.rb

```diff
@@ -2,4 +2,14 @@ class Task < ApplicationRecord
 
     belongs_to :user
     validates :description, presence: true
+
+    scheduler = Rufus::Scheduler.new
+
+    def self.update_task
+        Task.all.each { |task_obj| task_obj.update(is_completed: false) if task_obj.is_completed == true}
```

Comment:
Instead of manually looping through every single task and checking if each task is completed or not, we can make use of an `update_all` with a condition.

https://api.rubyonrails.org/v3.2.3/classes/ActiveRecord/Relation.html#method-i-update_all

Reply:
make sense, I didn't think about this method